From 5484110e3122efc646456fda8b3ca745e5dc91ae Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 4 Oct 2018 17:05:03 -0500
Subject: [PATH v4] Update default crio.conf for Clear Linux

Update paths to policy.json, seccomp.json, conmon and cni plugins
to reflect where they are intended to be used from Clear Linux.
Add docker.io as image registry. Set overlay as default storage
driver.

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---

Notes:
    Changes in v4: Support for several storage drivers was added,
            so trying to start crio service without a default produces
            errors. This change sets overlay as default storage driver
            in crio.conf

 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index 31d9ca7..b6cf08e 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,12 @@ bin/crio: .gopathok
 
 crio.conf: bin/crio
 	./bin/crio --config="" config --default > crio.conf
+	sed -i 's|^\(conmon = \).*|\1"/usr/libexec/crio/conmon"|' crio.conf
+	sed -i 's|^\(plugin_dir = \).*|\1"/usr/libexec/cni/"|' crio.conf
+	sed -i 's|^\(seccomp_profile = \).*|\1"/usr/share/defaults/crio/seccomp.json"|' crio.conf
+	sed -i "s|^\(#registries = \[\).*|registries = \[\n\t'docker.io',\n\]|" crio.conf
+	sed -i 's|^\(signature_policy = \).*|\1"/usr/share/defaults/crio/policy.json"|' crio.conf
+	sed -i 's|^\(#storage_driver = ""\)|storage_driver = "overlay"|' crio.conf
 
 release-note:
 	@$(GOPATH)/bin/containerd-release -n $(release)
-- 
2.19.0

